src/universal/components/MeetingHelp/HelpMenuBody.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'react-emotion'
+
+const HelpMenuBody = styled('div')({
```

Comment:
replaced the `& p` css with its own component.
in the bad old days, i remember wondering why on earth a child component would had a certain style. Almost always it's because its parent (or grandparent) had some trickle-down css that broke the encapsulation